### DISCUSSION TITLE
umi4.0.7 antd pro 作为qiankun 主应用，各位配置怎么配的？

### DISCUSSION THREAD

**Discussion by DemoPan** [umijs/umi#8605] [Q&A] [answered]
**.umirc.ts**

**默认配置：** 
`export default defineConfig({});`

**qiankun相关配置要求：**
`export default {
  qiankun: {
    master: {},
  },
};`

**Comment by fz6m** [comment, marked as answer]
主应用的例子：

https://github.com/umijs/umi/blob/3c5913eafe6019d3c390831e01dfa9c5409882f7/examples/max/.umirc.ts#L83-L95

对应的子应用例子：

https://github.com/umijs/umi/blob/master/examples/qiankun-slave/.umirc.ts
